CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ endif()
 option(run_unittests "set run_unittests to ON to run unittests (default is OFF)" OFF)
 option(run_int_tests "set run_int_tests to ON to run the integration tests (default is OFF)" OFF)
 option(use_installed_dependencies "set use_installed_dependencies to ON to use installed packages instead of building dependencies from submodules" OFF)
-option(use_cppunittest "set use_cppunittest to ON to build CppUnitTest tests on Windows (default is ON)" OFF)
+option(use_cppunittest "set use_cppunittest to ON to build CppUnitTest tests on Windows (default is ON)" ON)
```

Comment:
>ON)" ON) [](start = 100, length = 8)

Actually why not change the comment to say default is OFF? Default being OFF is actually better as in most cases we do not want to build with use_cppunittest. Btw the build failed because of exactly this: CE something something is trying to build with use_cppunittest=ON and CPP UnitTest is not available.

That made me wonder if we're just opening up for going and fixing a bunch of configs on downstream repos for little ROI of having cpp unittest default on (which only makes sense on Windows anyway)

Reply:
i believe the test failed in the gate because the passed parameter is wrong. i can change that too. it's currently `-Duse_cppunittests:bool=OFF` with an added and unneeded 's'. but i agree that it should likely remain OFF as default. i will change them both